tests/test_spark.py

```diff
@@ -169,6 +216,17 @@ def _require_spark():
     _wait_for_spark()
 
 
+def _require_spark_cli():
```

Comment:
Oh, I see.  This function is necessary because `_require_spark` will not install the CLI if the marathon app for the package is already installed then, right?  If so, can you just move this functionality to _require_spark()?  I'd rather have that function just be fully responsible for installing the entire package, including the CLI.